tealish/base.py

```diff
@@ -1,35 +1,43 @@
+from typing import Any, List, Dict, Tuple, TYPE_CHECKING
 from tealish.errors import CompileError
 from .tealish_builtins import constants
 from .langspec import get_active_langspec
 
 
+if TYPE_CHECKING:
+    from . import TealWriter
+    from .expression_nodes import Constant
+    from .nodes import Block
+
 lang_spec = get_active_langspec()
 
-structs = {}
+structs: Dict[str, Dict[str, Any]] = {}
```

Comment:
Another dict type that should probably be at least a dict of some more concrete type